diff --git a/Makefile b/Makefile
index 7ea761d..acebce8 100644
--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ CUT = cut
 
 VERSION != $(GREP) VERSION version.h | $(CUT) -d \" -f2
 
-PREFIX = /usr/local
+PREFIX = /usr
 
 MANPREFIX.$(PREFIX)=$(PREFIX)/share/man
 MANPREFIX./usr/local=/usr/local/man
@@ -23,13 +23,10 @@ MANPREFIX.=/usr/share/man
 MANPREFIX=$(MANPREFIX.$(PREFIX))
 
 INCLUDES != pkg-config --cflags libnotify
-CFLAGS_EXTRA = -pedantic -Wall -Wextra -Werror -Wno-unused-parameter -Os
-CFLAGS := $(CFLAGS_EXTRA) $(INCLUDES) $(CFLAGS)
+CFLAGS := $(INCLUDES) $(CFLAGS)
 
 LIBS != pkg-config --libs libnotify
 LIBS := $(LIBS) -lm
-LDFLAGS_EXTRA = -s
-LDFLAGS := $(LDFLAGS_EXTRA) $(LDFLAGS)
 
 all: $(NAME) $(NAME).1
 
